[2023-03-30 18:35:14,188] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2023-03-30T18:30:00+00:00 [queued]>
[2023-03-30 18:35:14,204] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2023-03-30T18:30:00+00:00 [queued]>
[2023-03-30 18:35:14,205] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-03-30 18:35:14,205] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-03-30 18:35:14,205] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-03-30 18:35:14,237] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_aguardando> on 2023-03-30 18:30:00+00:00
[2023-03-30 18:35:14,245] {standard_task_runner.py:52} INFO - Started process 2805 to run task
[2023-03-30 18:35:14,252] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_aguardando', 'scheduled__2023-03-30T18:30:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpada__6kz', '--error-file', '/tmp/tmpun4rhgrm']
[2023-03-30 18:35:14,254] {standard_task_runner.py:80} INFO - Job 49: Subtask popula_elastic_aguardando
[2023-03-30 18:35:14,438] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2023-03-30T18:30:00+00:00 [running]> on host 20743fec282a
[2023-03-30 18:35:14,661] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_aguardando
AIRFLOW_CTX_EXECUTION_DATE=2023-03-30T18:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-30T18:30:00+00:00
[2023-03-30 18:35:14,684] {base.py:271} INFO - GET http://192.168.0.100:9200/ [status:200 request:0.019s]
[2023-03-30 18:35:16,754] {base.py:271} INFO - HEAD http://192.168.0.100:9200/_template/desk_aguardando [status:200 request:0.004s]
[2023-03-30 18:35:16,971] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2023-03-30 18:35:17,087] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.098s]
[2023-03-30 18:35:17,177] {python.py:173} INFO - Done. Returned value was: None
[2023-03-30 18:35:17,200] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_aguardando, execution_date=20230330T183000, start_date=20230330T183514, end_date=20230330T183517
[2023-03-30 18:35:17,257] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-30 18:35:17,316] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
